lib/core/src/chain_swap.rs

```diff
@@ -1218,9 +1247,22 @@ impl ChainSwapHandler {
         // Verify RBF
         let rbf_explicit = tx.input.iter().any(|tx_in| tx_in.sequence.is_rbf());
         if !verify_confirmation && rbf_explicit {
-            return Err(anyhow!("Transaction signals RBF"));
+            bail!("Transaction signals RBF");
         }
         // Verify amount
+        if chain_swap.payer_amount_sat > 0 {
```

Comment:
Perhaps this could be done in `verify_user_lockup_tx`?

Reply:
Good point! Addressed in [d990f87](https://github.com/breez/breez-sdk-liquid/pull/667/commits/d990f87ec36ca97af0c29570245930fa916def36)